luda-editor/luda-editor-client/src/app/sequence_list/mod.rs

```diff
@@ -170,8 +161,6 @@ fn calculate_sequence_duration(
                 .clips
                 .iter()
                 .fold(duration, |duration, clip| duration.max(clip.end_at)),
-            Track::Subtitle(track) => track.clips.iter().fold(duration, |duration, clip| {
-                duration.max(clip.end_at(namui::Language::Ko, subtitle_play_duration_measurer))
-            }),
+            Track::Subtitle(track) => Time::zero(),
```

Comment:
This is the break changes. I think it's not necessary to calculate duration of subtitle. If you think it's not a good idea, I can revert it.